recipes/cyclus/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+set -e
+
+if [ "$(uname)" == "Darwin" ]; then
+  # toolchain copy
+  export CC=clang
+  export CXX=clang++
+  export MACOSX_VERSION_MIN="10.9"
+  export MACOSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}"
+  export CMAKE_OSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}"
+  export CFLAGS="${CFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
+  export LDFLAGS="${LDFLAGS} -headerpad_max_install_names"
+  export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export LDFLAGS="${LDFLAGS} -lc++"
+  export LINKFLAGS="${LDFLAGS}"
+  export CFLAGS="${CFLAGS} -m${ARCH}"
+  export CXXFLAGS="${CXXFLAGS} -m${ARCH}"
+  # other
+  libext=".dylib"
+  export LDFLAGS="-rpath ${PREFIX}/lib ${LDFLAGS}"
+  export LINKFLAGS="${LDFLAGS}"
+  skiprpath="-DCMAKE_SKIP_RPATH=TRUE"
+
+  # Avoid accelerate
+  # If OpenBLAS is being used, we should be able to find the libraries.
+  # As OpenBLAS now will have all symbols that BLAS or LAPACK have,
+  # create libraries with the standard names that are linked back to
+  # OpenBLAS. This will make it easy for NumPy to find it.
+  test -f "${PREFIX}/lib/libopenblas.a" && \
+    ln -fs "${PREFIX}/lib/libopenblas.a" "${PREFIX}/lib/libblas.a"
+  test -f "${PREFIX}/lib/libopenblas.a" && \
+    ln -fs "${PREFIX}/lib/libopenblas.a" "${PREFIX}/lib/liblapack.a"
+  test -f "${PREFIX}/lib/libopenblas.${libext}" && \
+    ln -fs "${PREFIX}/lib/libopenblas.${libext}" "${PREFIX}/lib/libblas.${libext}"
+  test -f "${PREFIX}/lib/libopenblas.${libext}" && \
+    ln -fs "${PREFIX}/lib/libopenblas.${libext}" "${PREFIX}/lib/liblapack.${libext}"
```

Comment:
We should probably move this out of the `if`. I don't think this issue occurs with Linux as their shouldn't be any BLAS, but this could be handy for offline building where there might be.